packaged_releases/HISTORY.md

```diff
@@ -1,6 +1,59 @@
 Release History
 ===============
 
+2.0.8 (2017-06-13)
```

Comment:
@derekbekoe do we need to add it here? since we didn't ship it in neither bundle or apt.